libs/perun/pipes/src/lib/member-status-tooltip.pipe.ts

```diff
@@ -23,17 +23,28 @@ export class MemberStatusTooltipPipe implements PipeTransform {
     if (showGroupStatuses) {
       res = `\n Group status: ${parseMemberStatus(member.groupStatus)}, Expiration: ${
         (groupExpiration?.value as string) ?? 'never'
+      } ${
+        isMemberIndirect(member)
```

Comment:
Since this is added for each case, do this check once at the end to avoid code duplication.